Source/WebCore/workers/service/server/SWServerWorker.cpp

```diff
@@ -177,7 +177,7 @@ const ClientOrigin& SWServerWorker::origin() const
     return *m_origin;
 }
 
-SWServerToContextConnection* SWServerWorker::contextConnection()
+RefPtr<SWServerToContextConnection> SWServerWorker::contextConnection()
```

Comment:
I think it is ok to have a getter return a RefPtr if that makes the code more readable, we do not need `protected` since there is no unprotected getter. I'll change back to return a raw pointer.